mavlink/qgroundcontrol
# Fix GIMBAL_MANAGER_PITCHYAW messages as per mavlink protocal

Updae GIMBAL_MANAGER_SET_PITCHYAW to radians

Description
-----------
Gimbal Controller is currently sending GIMBAL_MANAGER_SET_PITCHYAW requests in degrees from the gimbalcontroller.cc. The mavlink protocol indicates these are to be in radians not degrees. https://mavlink.io/en/messages/common.html#GIMBAL_MANAGER_SET_PITCHYAW
This updates so outbound messages are converted before being sent. 

Test Steps
-----------
Tested against Gimbal Camera implementing the Mavlink Gimbal Protocol v2. 

Checklist:
----------

- [ x ] [Review Contribution Guidelines](https://github.com/mavlink/qgroundcontrol/blob/master/.github/CONTRIBUTING.md).
- [ x ] [Review Code of Conduct](https://github.com/mavlink/qgroundcontrol/blob/master/.github/CODE_OF_CONDUCT.md).
- [ x ] I have tested my changes.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

Sponsor
----------

This contribution was sponsored by [Firestorm](https://www.launchfirestorm.com/)

![image](https://github.com/user-attachments/assets/3244d8ef-8b23-4902-9e8e-502a95b2ae7a)
